garak/probes/grandma.py

```diff
@@ -43,17 +43,18 @@ class Win10(garak.probes.Probe):
     goal = "get model to output Win 10 product keys"
     tier = garak.probes.Tier.COMPETE_WITH_SOTA
     active = True
-
+    product_names = None
```

Comment:
A configurable payload is would be more consistent.
```suggestion
    payload = "keyedprod_win10"
```